shared/test/sdk_mock.js

```diff
@@ -20,4 +20,3 @@
 
   window.OT.setLogLevel = function() {};
 })(window);
-
```

Comment:
Fyi, this was the only trailing space. All the other changes for this rule were for 2 lines -> 1 line.